wgpu/src/lib.rs

```diff
@@ -4531,3 +4582,35 @@ impl Display for Error {
         }
     }
 }
+
+use send_sync::*;
+
+mod send_sync {
+    use std::any::Any;
+    use std::fmt;
+
+    use wgt::{WasmNotSend, WasmNotSync};
+
+    pub trait AnyWasmNotSendSync: Any + WasmNotSend + WasmNotSync {
```

Comment:
I left this trait here because it's only used in this crate.
I hope the name isn't too awful.